clients/drcachesim/common/options.cpp

```diff
@@ -920,6 +920,14 @@ droption_t<bool> op_sched_randomize(
     "set), and FIFO order and instead selects the next input randomly. "
     "This is intended for experimental use in sensitivity studies.");
 
+droption_t<bool> op_sched_disable_direct_switches(
```

Comment:
I suppose it was on purpose that this is a "disable" flag? Generally "enable" flags are easier for humans to read and parse. Optional: would it help if it were sched_enable_direct_switches, with default set to true?

Reply:
OTOH it is natural for humans to think that specifying an option changes the default: so if it's named --enable it sounds like it's not enabled unless the flag is specified.  The other factor is when options like this are stored in structures like protocol buffers which sometimes do not have an easy non-zero default, where having 0 be the default is just easier to avoid having differing defaults in command-line options versus unit test tools using structures.

Reply:
Continuing on the protocol buffer discussion: having that variation in defaults has caused headaches in the past.

Reply:
SGTM